src/test/java/org/folio/rest/impl/EndToEndTest.java

```diff
@@ -279,7 +279,7 @@ private void givenSetSourceStorageMockToReturnRecords() throws IOException {
   private Future<Object> assertSuccessJobExecution(TestContext context, FileDefinition fileDefinition,  Optional<JobExecution> jobExecutionOptional, Integer currentNumber, String total) {
     JobExecution jobExecution = jobExecutionOptional.get();
     context.assertTrue(isFileNameContainsJobExecutionHrId(new File(fileDefinition.getSourcePath()).getName(), jobExecution.getHrId()));
-    context.assertEquals(jobExecution.getStatus(), SUCCESS);
+    context.assertEquals(jobExecution.getStatus(), COMPLETED);
```

Comment:
Please add a test case for "completed with errors"

Reply:
Please take care of this @Igor-Gorchakov

Reply:
The test is in place

Reply:
Thanks!